src/components/upload/FileUpload.js

```diff
@@ -1,36 +1,44 @@
-import styled from "styled-components";
 import FileDropZone from "./FileDropZone";
-import {useState} from "react";
 import Settings from "../common/settings";
-import Notifications from "../common/Notifications";
 import {Confirmation} from "../common/StyledComponents";
 import FetchHandler from "../common/FetchHandler";
+import {styled} from '@mui/system';
+import Input from '@mui/material/Input';
+import UploadIcon from '@mui/icons-material/Upload';
+import {useState} from "react";
+import EditorPage from "../editor/EditorPage";
+import Box from '@mui/material/Box';
+import * as React from 'react';
+import {Button,TextField,Dialog} from '@mui/material';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
 
-
-const FileUploadRoot = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-`;
-
-const Input = styled.input`
-  height: 2em;
-  border-radius: 4px;
-  border: none;
-  margin-right: 20px;
-`;
-
-const UploadButton = styled.button`
-  height: 2em;
-  width: 10em;
-`;
-
+/*
+  const UploadButton = styled(Button)({
+    background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+    borderRadius: 3,
+    height: 48,
+    padding: '0 30px',
+  }); */
+  const FileUploadRoot = styled('div')({
+    alignItems: "center",
+    boxShadow: '0 3px 5px 2px ',
+    background: '#FFFFFF',
+    padding: '0 20px',
+    
+  });
```

Comment:
```suggestion
const FileUploadRoot = styled('div')({
    boxShadow: '0 3px 5px 2px',
    padding: '20px'
});
```
`alignItems` and `background` have no effect here.
`padding` should be at the top and bottom too.
Also: Fix the indentation